mappings/net/minecraft/entity/ai/goal/PitchChangingJumpingGoal.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS ajw net/minecraft/entity/ai/goal/PitchChangingJumpingGoal
```

Comment:
Maybe something like `DiveJumpGoal`?

Reply:
This is the higher-than-one-block jump shared by foxes and dolphins. Is this dive jump?

Reply:
I'd think the name applies well enough to both

Reply:
Done.